docs/install/index.md

```diff
@@ -3,36 +3,29 @@ html_meta:
   "description": "Install Plone 6"
   "property=og:description": "Install Plone 6"
   "property=og:title": "Install Plone 6"
-  "keywords": "Plone 6, install, installation, installing, buildout, pip, docker"
+  "keywords": "Plone 6, install, overview"
 ---
 
 (install-index-label)=
 
 # Install Plone 6
 
-This section focuses on installing Plone 6 for the developer, tester, or contributor.
 
-For trying out Plone 6, visit our official demonstration site located at:
+(install-index-getting-started-label)=
 
-```{toctree}
-:maxdepth: 2
-:hidden: true
+## Getting started
 
-containers/index
-```
+* What do you want to do?
+  * Try a demo now
+  * Run Plone in containers (why use containers here..)
```

Comment:
Link to chapter "Plone with docker"

Reply:
Fixed in 2cd2f50e0ae40ed6950d22405e27d06cd1a568d8